src/main/kotlin/at/bitfire/dav4jvm/HttpHeaders.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright © All Contributors. See LICENSE and AUTHORS in the root directory for details.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package at.bitfire.dav4jvm
+
+object HttpHeaders {
+    val ETag: String = "ETag"
+    val ScheduleTag: String = "Schedule-Tag"
```

Comment:
1. I think we should find a better name for the class. In `HttpHeaders` I expected some common interface for HTTP headers and not specific HTTP header names. Maybe something like `HttpHeaderNames`. We can then define the constants and mention where they are defined (RFC reference) in KDoc.
2. If we decided to use such a class, we should put other headers in there too. There are at least "Accept" and "Accept-Encoding".
3. We can make those `const val`.